src/components/TodoForm.vue

```diff
@@ -1,144 +1,327 @@
+  togglePopup();
+}
+function saveTodo(toDo: Todotype) {
+  toDo.isEditing = false;
+  console.log("saveTodo");
+}
+onMounted(() => {
+  const storedTodos = localStorage.getItem("todos");
+  if (storedTodos !== null) {
+    todos.value = JSON.parse(storedTodos);
+  }
+});
+
+watch(
+  todos,
+  (newValue) => {
+    localStorage.setItem("todos", JSON.stringify(newValue));
+  },
+  { deep: true }
+);
 </script>
 
 <style scoped>
-.no-todos-text {
-  font-family: "Neue Haas Grotesk Display Pro";
-  font-style: normal;
-  font-weight: 600;
-  font-size: 42px;
-  line-height: 50px;
-
-  color: #6d6d6d;
-}
-.todo-list {
-  margin-top: 170px;
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Neue Hass Grotesk Display Pro";
 }
-.prio-check {
+.no-input {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
-  align-items: flex-end;
+  align-items: center;
+}
+.workflow-img {
+  width: 192px;
+  height: 221px;
 }
-.delete-btn {
+
+.text-priority {
   display: flex;
-  flex-direction: column;
+  align-items: center;
   justify-content: space-between;
+
+  flex-grow: 1;
 }
-.list {
+
+.content-container {
   display: flex;
-  flex-direction: column;
   align-items: center;
-  gap: 50px;
+  width: 100%;
+}
+.status {
+  margin-right: 18px;
 }
-
 .listed-items {
-  padding: 20px;
   display: flex;
-  justify-content: space-between;
-  box-sizing: border-box;
-  list-style: none;
-
-  background: #ffffff;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  position: relative;
+  min-height: 82px;
+  width: 280px;
   border: 2px solid #000000;
   border-radius: 16px;
-  width: 600px;
-  height: 163px;
 }
+
+.content {
+  position: relative;
+  flex-direction: row-reverse;
+  height: 100%;
+  width: 100%;
+}
+.todo-text{
+  border: 0;
+}
+
+.text-title {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+}
+
+.title {
+  font-family: "SF Pro Text";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  width: 70%;
+  border: 0;
+}
+
+.delete-button {
+  font-size: 14px;
+  font-weight: 600px;
+  line-height: 17px;
+  width: 69px;
+  height: 27px;
+  border-radius: 8px;
+  border: 0;
+  outline: 0;
+  background-color: #e5e5e5;
+  padding: 5px 15px;
+}
+
+.delete-save-btn {
+  display: flex;
+  gap: 10px;
+  width: 100%;
+  margin-top: 38px;
+}
+
+.buttons {
+  width: 60px;
+  height: 27px;
+}
+
+.no-todos-text {
+  font-size: 22px;
+  line-height: 26px;
+}
+
 .workflow-img {
-  width: 270px;
-  height: 315px;
+  width: 190px;
+  height: 220px;
 }
 
-@media (max-width: 768px) {
-  .no-todos-text {
-    font-size: 22px;
-    line-height: 26px;
-  }
-  .workflow-img {
-    width: 190px;
-    height: 220px;
-  }
-  .listed-items {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px;
+.todo-list {
```

Comment:
Please review the styles and merge the common styles. Ex. `list` class contains the `.todo-list`.  You don't need to set the same style more times. Please try not to duplicate the code.